src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -113,89 +113,103 @@
 		<template #list>
 			<li ref="container" class="left-sidebar__list" @scroll="debounceHandleScroll">
 				<ul class="scroller">
-					<NcListItem v-if="noMatchFound && searchText"
-						:title="t('spreed', 'Create a new conversation')"
-						@click="createConversation(searchText)">
-						<template #icon>
-							<ChatPlus :size="30" />
-						</template>
-						<template #subtitle>
-							{{ searchText }}
-						</template>
-					</NcListItem>
-
-					<NcAppNavigationCaption :class="{'hidden-visually': !isSearching}"
+					<NcAppNavigationCaption v-if="!isSearching"
+						class="hidden-visually"
```

Comment:
It's always hidden from the DOM now. Maybe remove NcAppNavigationCaption from here at all?